examples/00-load/read-image.py

```diff
@@ -32,3 +33,29 @@
 
 # Mapped image colors
 image.plot(cpos="xy")
+
+###############################################################################
+# Convert rgb to grayscale.
+# https://en.wikipedia.org/wiki/Grayscale#Luma_coding_in_video_systems
+
+r = image["JPEGImage"][:, 0]
+g = image["JPEGImage"][:, 1]
+b = image["JPEGImage"][:, 2]
+image.clear_data()
+image["GrayScale"] = 0.299 * r + 0.587 * g + 0.114 * b
+pv.global_theme.cmap = "gray"
+image.copy().plot(cpos="xy")
```

Comment:
Why are these copies necessary for plotting?

Reply:
Explanation provided.